test/fixtures/parser/mysql/force_index.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+       onetable.f1,
+       twotable.f1
+FROM onetable
+         LEFT JOIN
+     twotable
+         FORCE INDEX FOR JOIN (idx_index)
+     ON onetable.f1 = twotable.f1
```

Comment:
Thanks for contributing this @Aeolun 🙇, could you also add a .yml file in here per https://github.com/sqlfluff/sqlfluff/blob/master/test/fixtures/parser/README.md to demonstrate the new parsing behaviour and result?